README.md

```diff
@@ -383,47 +386,76 @@ Which results in the following output:
 ### Working with Generic ASN.1 Structures
 
 The magic shown above is based on a from-scratch 100% KMP implementation of an ASN.1 encoder and parser.
-To parse any DER-encoded ASN.1 structure, call `Asn1Element.parse(derBytes)`, which will result in exactly a single
-`Asn1Element`.
-It can be re-encoded (and yes, it is a true re-encoding, since the original bytes are discarded after decoding) by
-accessing the lazily evaluated `.derEncoded` property.
+To parse any DER-encoded ASN.1 structure, call either:
+
+* `Asn1Element.parse(derBytes)`, which will consume all bytes and return the first parsed ASN.1 element.
+This method throws if more than a single toplevel ASN.1 Element it found or if any parsing errors occur.
```

Comment:
so "if there are any trailing bytes after the element", right?

Reply:
true. optimized the code in that respect too.